packages/components/src/components/hds/table/index.ts

```diff
@@ -50,11 +51,12 @@ export interface HdsTableArgs {
     isFixedLayout?: boolean;
     isSelectable?: boolean;
     isStriped?: boolean;
-    model: Array<Record<string, unknown>>;
+    model: HdsTableModel;
```

Comment:
suggestion: to ease the rebase with #2421 (marking the model as optional)
```suggestion
    model?: HdsTableModel;
```

Reply:
Agreed